### ISSUE TITLE
[publish-features] released feature doesn't point to the feature README, but repo README

### ISSUE THREAD

**Issue by snebjorn** [devcontainers/action#82]
A release feature points to the README at the root repository.
See https://github.com/snebjorn/devcontainer-feature/pkgs/container/devcontainer-feature%2Fchromium

I was expecting it to point to the README inside the `src/<feature-name>` folder.
Being this one https://github.com/snebjorn/devcontainer-feature/blob/main/src/chromium/README.md for the above mentioned feature.

**Comment by joshspicer** [comment]
Agreed that would be cool.  This will require a change with GitHub packages itself to allow us to define an arbitrary README path per Feature.
